websites/D/Doronime/presence.ts

```diff
@@ -8,96 +8,119 @@ const presence = new Presence({
 	};
 
 presence.on("UpdateData", async () => {
+	const capitalize = (str: string[]) =>
+		str.map(x => x[0].toUpperCase() + x.slice(1)).join(" ");
```

Comment:
Let's also add a way to trim e.g.
```suggestion
	const capitalize = (str: string[]) =>
		str.map(x => x[0].toUpperCase() + x.slice(1)).join(" "),
		trim = (str: string[]) =>
		str.trim();
```
![image](https://github.com/PreMiD/Presences/assets/42322979/78dc4bfc-33d5-4cf9-9e16-0255eabd4f6e)